presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -452,6 +452,9 @@ public void testSTIntersects()
         assertFunction("ST_Intersects(ST_GeometryFromText('MULTILINESTRING ((1 1, 5 1), (2 4, 4 4))'), ST_GeometryFromText('MULTILINESTRING ((3 4, 6 4), (5 0, 5 4))'))", BOOLEAN, true);
         assertFunction("ST_Intersects(ST_GeometryFromText('POLYGON ((1 1, 1 3, 3 3, 3 1))'), ST_GeometryFromText('POLYGON ((4 4, 4 5, 5 5, 5 4))'))", BOOLEAN, false);
         assertFunction("ST_Intersects(ST_GeometryFromText('MULTIPOLYGON (((1 1, 1 3, 3 3, 3 1)), ((0 0, 0 2, 2 2, 2 0)))'), ST_GeometryFromText('POLYGON ((0 1, 3 1, 3 3, 0 3))'))", BOOLEAN, true);
+        assertFunction("ST_Intersects(ST_GeometryFromText('LINESTRING (16.6 54.0167, 16.6 54.017)'), ST_GeometryFromText('LINESTRING (16.6 53, 16.6 56)'))", BOOLEAN, true);
```

Comment:
Is there any particular reason to add these tests? It seems that tests on lines 450 and 452 cover linestrings already.

Reply:
Oh damn. It is just a copy-paste error. Instead of first `LINESTRING` i was going to paste a polygon. Thanks for the catch.

So basically i was going to cover the cases used in benchmark. Like line that crosses the polygon. Line that lies out of the polygon, but crosses the envelope. And a line that lies down out of the polygon's envelope. All the existing test cases that make `intersects` to return false are based on the geometries that exactly match their envelope.